x-pack/plugins/security_solution/common/api/detection_engine/prebuilt_rules/model/diff/diffable_rule/diffable_rule.ts

```diff
@@ -13,7 +13,6 @@ import {
   HistoryWindowStart,
   InvestigationFields,
   InvestigationGuide,
```

Comment:
Kibana doesn't accept requests to internal API endpoints starting from `v9.0`. To allow such requests add `server.restrictInternalApis: false` to `kibana.dev.yaml`.

Sorry, I forgot to mention that since have that flag enable for quite some time.